content/help/addons/hero-image.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Hero Image
+tags:
+- Addons
+- Hero Image
+- Image upload
+category: help
+menu:
+  help:
+    parent: addons
+    identifier: addon-hero-image
+versioning:
+  added: 2.6
+---
+
+## Overview
+
+{{% cloudfeature %}}
+
+This plugin allows you to:
+
+- Associate an background image with a category.
+- Inherit the parent categories image if one isn't set.
+- Set a "top-level" image that everything falls back to if no image is found on a parent category.
+- Access this image through a [Smarty Template]() or [Customize Theme]().
+
+## Setup
+
+- Visit the addon's page of the dashboard.
+- Enable the HeroImage plugin.
```

Comment:
You say "enable" here, but your screen cap has the cursor on the settings page. I'd put another screen shot, as obvious as it may seem to us.